weave-js/src/components/Sidebar/Outline.tsx

```diff
@@ -259,7 +260,7 @@ const OutlinePanel: React.FC<OutlinePanelProps> = props => {
 };
 
 export interface OutlineProps {
-  config: ChildPanelFullConfig;
+  config: ChildPanelFullConfig<PanelGroupConfig>;
```

Comment:
this helps make type checking more robust & hopefully improves dev exp a little bit!